testing/file_test/file_test_base.h

```diff
@@ -148,12 +132,29 @@ class FileTestBase : public testing::Test {
   static auto TransformExpectation(int line_index, llvm::StringRef in)
       -> testing::Matcher<std::string>;
 
-  const std::filesystem::path* path_;
+  const std::filesystem::path path_;
+};
+
+struct FileTestFactory {
+  // The test fixture name.
+  const char* name;
```

Comment:
I'm guessing `llvm::StringLiteral` doesn't work here? Unfortunate, but understandable if so.

Reply:
It just gets awkward:

```
Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
testing/file_test/file_test_base.cpp:368:5: error: no matching function for call to 'RegisterTest'
    testing::RegisterTest(test_factory.name, test_name.c_str(), nullptr,
    ^~~~~~~~~~~~~~~~~~~~~
external/com_google_googletest/googletest/include/gtest/gtest.h:2259:11: note: candidate function template not viable: no known conversion from 'llvm::StringLiteral' to 'const char *' for 1st argument
TestInfo* RegisterTest(const char* test_suite_name, const char* test_name,
          ^
```